picard/ui/options/general.py

```diff
@@ -41,6 +41,7 @@ class GeneralOptionsPage(OptionsPage):
         config.TextOption("persist", "oauth_refresh_token", ""),
         config.BoolOption("setting", "analyze_new_files", False),
         config.BoolOption("setting", "ignore_file_mbids", False),
+        config.BoolOption("setting", "cluster_new_files", False),
```

Comment:
I think we should leave it to False for 1.4 and introduce all these "automatic" changes from further versions onwards.